ports/espressif/boards/adafruit_feather_esp32_v2/pins.c

```diff
@@ -15,6 +15,7 @@ static const mp_rom_map_elem_t board_module_globals_table[] = {
 
     { MP_ROM_QSTR(MP_QSTR_A1), MP_ROM_PTR(&pin_GPIO25) },
     { MP_ROM_QSTR(MP_QSTR_D25), MP_ROM_PTR(&pin_GPIO25) },
+    { MP_ROM_QSTR(MP_QSTR_SPEAKER), MP_ROM_PTR(&pin_GPIO25) },
```

Comment:
I think it's our practice with adafruit boards to only add `board.SPEAKER` when there is a speaker on the board. It's not merely added when a pin is capable of analog output. Please remove the line.